### PR TITLE
allowing to handle 404 in case of a file not found in fpm

### PR DESCRIPTION
**Pull request by venturel**

### CODE REVIEW

File: templates/etc/nginx/sites-available/fpm.conf.erb [hunk 1]
```diff
@@ -41,7 +41,8 @@ server {
 
   location ~ ^(.+\.php)(.*)$ {
     include fastcgi_params;
-
+    #allows to handle 404 in case of a file not found in fpm
+    try_files $uri =404;
```
Comment (MattiasGees) [inline templates/etc/nginx/sites-available/fpm.conf.erb]: I think it is best to not do it by default. Most php applications don't have it implemented.

Comment (venturel) [inline templates/etc/nginx/sites-available/fpm.conf.erb]: What this line says is to reply with a 404 if it doesn't find the file with the default nginx error page. 

Currently if it doesn't find the php page it replies with File not found.

Comment (duboisph) [inline templates/etc/nginx/sites-available/fpm.conf.erb]: Indeed, the `=404` means the default 404 page of nginx. So if the application can not handle it itself, nginx should do so.

**Comment by duboisph** [comment]
👍

**Comment by MattiasGees** [comment]
👍

**Pull request merged**